scripts/core/interactive-article-actions-panel/actions/publish-tab.tsx

```diff
@@ -159,13 +160,26 @@ export class PublishTab extends React.PureComponent<IProps, IState> {
                     <div style={style}>
                         <div>
                             {
-                                publishFromEnabled && (
+                                true && (
```

Comment:
did you review your code yourself before opening the PR? :upside_down_face:

Reply:
Yes, missed this one